### ISSUE TITLE
Ride Summary CP Wbal?

### ISSUE THREAD

**Issue by bdb** [GoldenCheetah/GoldenCheetah#1776]
Reviewing code again and it appears that CP Wbal was started on but never finished.

RideSummaryWindow:2286 declares CP wbal vars, but then those are never used for output summary. Are they for future work or just leftovers that should be removed?

**Comment by liversedge** [comment]
Yes. The time in W'bal zone is on an activity only at present .. we should probably fix that !
BTW, that line of code is for compare mode. When compare mode was implemented some of the charts got really nasty -- 4 scenarios to support;
1. plot ride
2. plot ride comparisons
3. plot for a date range
4. plot for comparison of date range

In some cases that was just a simple case of using arrays/lists and in others (summary is a good example) it mean't changing the outputs and inputs. Due to time (it took 3 months to implement compare mode) we ended up avoiding refactoring some charts where that was needed.

There is an open FR to rewrite summary window .............
